restructuredtext_lint/cli.py

```diff
@@ -1,46 +1,54 @@
 import argparse
 import json
 import sys
+import locale
 
 from .lint import lint_file
 
 
-def _main(filepath, format='text', stream=sys.stdout, encoding=None):
+def _main(filepaths, format, encoding, stream=sys.stdout):
     # Read and lint the file
-    errors = lint_file(filepath, encoding=encoding)
+    file_errors = [lint_file(f, encoding=encoding) for f in filepaths]
+    errors_dicts = []
 
-    # If there were no errors, exit gracefully
-    if not errors:
-        if format == 'json':
-            stream.write('[]')
+    for filename, errors in zip(filepaths, file_errors):
+        if not errors:
+            stream.write('INFO File {} is clean.\n'.format(filename))
```

Comment:
Ah, I see where the 'INFO' is coming from. Never mind leave that in but I would still like the kwarg `.format`